### DISCUSSION TITLE
Sanmill v1.0.25

### DISCUSSION THREAD

**Discussion by calcitem** [calcitem/Sanmill#230] [General]
Add EULA link.

<hr /><em>This discussion was created from the release <a href='https://github.com/calcitem/Sanmill/releases/tag/v1.0.25'>Sanmill v1.0.25</a>.</em>

**Comment by calcitem** [comment]
From June 18, 2021, it will be available for download in English and German-speaking countries on the Play Store.

**Comment by justinacolmena** [comment]
Very traditional game. Three pieces in a row,
"I'll just take a mil here."
Or, ...
"You know, we placed nine men on the school board there, but we lost a mil when that real estate deal fell through."

"Mil" was slang for $1,000,000

**Reply by calcitem** [reply]
Hi!  Can you tell me what I need to do?  Do I need to fix the help description or where is the spelling mistake? Or it is only a joke?

**Reply by justinacolmena** [reply]
Oh no. It's all part of the game.

**Reply by calcitem** [reply]
I currently don't understand what I need to do.  Could you describe the problem in more detail?  Sorry for my bad English.

**Comment by justinacolmena** [comment]
Finns, Swedes, Sami, Norwegians, Russians, etc. played this board game in northern Europe many many years ago.

There was a lot of strategy to the play and high stakes were involved.

Capturing an enemy's piece was referred to as "taking a mill" if I remember. Don't take me at my word for it, though.

On July 25, 2021 7:58:53 AM AKDT, Calcitem ***@***.***> wrote:
>I currently don't understand what I need to do.  Could you describe the
>problem in more detail?  Sorry for my bad English.
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/calcitem/Sanmill/discussions/230#discussioncomment-1047655

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by calcitem** [comment]
Although this App has machine translation in the languages ​​of the Nordic countries, there are not many downloads at present, and the reason is unknown.

Do I need to change the string "removing a mill" to "taking a mill"?

**Comment by justinacolmena** [comment]
No. "Taking" is always capturing.

I'm trying to remember, because you "form a mill" by placing three of your own pieces in a row — but then you "take a mill" by removing only one of your enemy's pieces.

A mill is "kvarn" in Swedish and "mylly" in Finnish (like the German). It was highly illegal in Sweden back then, and a playing piece would have been called a "kvarnsten" (in the sense of a casino chip or gambling token) but you couldn't use that word or refer to it directly because the police would bust the game even if no money was involved.



On July 25, 2021 9:03:22 AM AKDT, Calcitem ***@***.***> wrote:
>Although this App has machine translation in the languages ​​of the
>Nordic countries, there are not many downloads at present, and the
>reason is unknown.>
>>
>Do I need to change the string "moving a mill" to "taking a mill"?>
>________________________________>
>From: justina colmena ***@***.***>>
>Sent: Monday, July 26, 2021 12:53:51 AM>
>To: calcitem/Sanmill ***@***.***>>
>Cc: Calcitem ***@***.***>; Author ***@***.***>>
>Subject: Re: [calcitem/Sanmill] Sanmill v1.0.25 (#230)>
>>
>>
>Finns, Swedes, Sami, Norwegians, Russians, etc. played this board game
>in northern Europe many many years ago.>
>>
>There was a lot of strategy to the play and high stakes were involved.>
>>
>Capturing an enemy's piece was referred to as "taking a mill" if I
>remember. Don't take me at my word for it, though.>
>>
>On July 25, 2021 7:58:53 AM AKDT, Calcitem ***@***.***> wrote:>
>>I currently don't understand what I need to do. Could you describe
>the>
>>problem in more detail? Sorry for my bad English.>
>>>
>>-->
>>You are receiving this because you commented.>
>>Reply to this email directly or view it on GitHub:>
>>https://github.com/calcitem/Sanmill/discussions/230#discussioncomment-1047655>
>>
>-->
>Sent from my Android device with K-9 Mail. Please excuse my brevity.>
>>
>—>
>You are receiving this because you authored the thread.>
>Reply to this email directly, view it on
>GitHub<https://github.com/calcitem/Sanmill/discussions/230#discussioncomment-1047761>,
>or
>unsubscribe<https://github.com/notifications/unsubscribe-auth/AMJCRESN52XI6KSXPPBSJJLTZQ6R7ANCNFSM463ZS7FA>.>
>>
>>
>-- >
>You are receiving this because you commented.>
>Reply to this email directly or view it on GitHub:>
>https://github.com/calcitem/Sanmill/discussions/230#discussioncomment-1047773

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Reply by calcitem** [reply]
In Sweden, is it filtered out because the translation contains illegal words?

**Comment by justinacolmena** [comment]
That's possible. It was also a property tax thing at a time when peasant farmers revolted in Europe and Russia because they were strictly forbidden from possessing their own mills, but required to deliver their grain to the county mill and have it inspected and graded by the county miller under the direction of an earl or count or duke.

Even at the point of bloody revolution, property taxes were a lot lower back then and assessed "per mille" (‰) — nowadays they are much higher and assessed by "percent" (%).

On July 25, 2021 9:42:38 AM AKDT, Calcitem ***@***.***> wrote:
>In Sweden, is it filtered out because the translation contains illegal
>words?
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/calcitem/Sanmill/discussions/230#discussioncomment-1047850

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
